pandas/tests/io/test_spss.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas.util import testing as tm
+
+from pandas.io.spss import read_spss
+
+try:
+    import pyreadstat  # noqa
```

Comment:
use ``pyreadstat = pytest.importorskip('pyreadstat')``; no need for the skipifs

Reply:
Done.